packages/google-drive-ocamlfuse/google-drive-ocamlfuse.0.6.5/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Alessandro Strada <alessandro.strada@gmail.com>"
+authors: [ "Alessandro Strada" ]
+license: "MIT"
+homepage: "http://gdfuse.forge.ocamlcore.org/"
+dev-repo: "https://github.com/astrada/google-drive-ocamlfuse.git"
+bug-reports: "https://github.com/astrada/google-drive-ocamlfuse/issues"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocaml" "%{etc}%/google-drive-ocamlfuse/_oasis_remove_.ml"
+    "%{etc}%/google-drive-ocamlfuse"]
+]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+depends: [
+  "base-threads" {build}
+  "camlidl" {build}
+  "gapi-ocaml" {build & >= "0.2.14"}
```

Comment:
I would suggest to remove the `build` tags for things that you want to see trigger a recompilation of your programs. @Chris00 what is the rational for adding `build` everywhere here?